### PR TITLE
Added aggregate login switch

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-1605](https://team-1624093970686.atlassian.net/browse/AR-1605).

## Blocking Dependencies

[AR-1895](https://team-1624093970686.atlassian.net/browse/AR-1895)

## Changes

1.  Added a switch that allows developers to choose aggregate login in auth type
2. Switch is active when creating the app and disabled after creation of app (Developers can't change this setting after the app is created)
3. Added sample text in tooltip untils AR-1895 is resolved

## Screenshots

Screenshot from 2022-04-20 15-26-26

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: src/components/app-configure/AppAuth.vue [hunk 1]
```diff
@@ -28,7 +28,21 @@
...
       </div>
       <div class="flex column">
-        <div class="flex input-group">
+        <div class="flex" style="gap: 0.625rem">
+          <span class="body-1" style="color: var(--text-grey); font-family: var(--font-title);">
+            Aggregate Login
+          </span>
+          <v-switch :value="hasAggregateLogin" @checked="updateAggregateLogin" :disabled="isConfigured" style="margin-top: 1px"></v-switch>
+          <v-tooltip
+            title="Aggregate Login (Needs content here)"
```
Comment (ajithranka) [inline src/components/app-configure/AppAuth.vue:37]: Who needs to provide this content?

File: src/components/app-configure/AppAuth.vue [hunk 2]
```diff
@@ -157,6 +171,7 @@ import VDropdown from "@/components/lib/VDropdown/VDropdown.vue";
 import VIconButton from "@/components/lib/VIconButton/VIconButton.vue";
 import VTextField from "@/components/lib/VTextField/VTextField.vue";
 import VTooltip from "@/components/lib/VTooltip/VTooltip.vue";
+import VSwitch from "../lib/VSwitch/VSwitch.vue";
```
Comment (ajithranka) [inline src/components/app-configure/AppAuth.vue:174]:
```suggestion
import VSwitch from "@/components/lib/VSwitch/VSwitch.vue";
```

File: src/components/app-configure/AppAuth.vue [hunk 3]
```diff
@@ -234,6 +250,7 @@ export default {
     let selectedAuthOrigin = ref("");
     let selectedAuthRedirectUrl = ref("");
     let errorMessage = ref("");
+    let hasAggregateLogin = ref(store.getters["test/hasAggregateLogin"]);
```
Comment (ajithranka) [inline src/components/app-configure/AppAuth.vue:253]: Shouldn't this be a computed property?

**Review by ajithranka — APPROVED** [review]

File: src/components/app-configure/AppAuth.vue (hunk 1 above)
Comment (shrinathprabhu) [reply src/components/app-configure/AppAuth.vue:37]: Anirudh, there is a ticket, I have added the blocking dependency

File: src/components/app-configure/AppAuth.vue (hunk 3 above)
Comment (shrinathprabhu) [reply src/components/app-configure/AppAuth.vue:253]: Yeah

**Review by karthik-durai — APPROVED** [review]

**Pull request merged**